Lib/test/test_source_encoding.py

```diff
@@ -305,5 +295,18 @@ def check_script_output(self, src, expected):
         self.assertEqual(res.out.rstrip(), expected)
 
 
+class ImportTests(unittest.TestCase):
+    def setUp(self):
+        mod = importlib.import_module('test.encoded_modules')
+        self.test_strings = mod.test_strings
+        self.test_path = mod.__path__
+
+    def test_import_encoded_module(self):
```

Comment:
You can also remove this from `test_imp` (a breadcrumb comment pointing back to here might not be amiss, though).